src/index.html

```diff
@@ -1,18 +1,76 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="card"
+  data-qa="card"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <header class="header">
+      <img
+        class="header__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+    </header>
+
+    <main class="main">
+      <h1 class="main__product-name">
+        APPLE A1419 iMac 27&#8220; Retina 5K Monoblock (MNED2UA&#47;A)
+      </h1>
+
+      <h2 class="main__product-code">Product code: 195434</h2>
+
+      <container class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+
+        <div class="reviews">Reviews: 5</div>
+      </container>
+
+      <div class="price">
```

Comment:
there should be only 2 BEM blocks card and stars. Others are just elements of these blocks